hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -869,7 +874,8 @@ private synchronized void flushWrittenBytesToServiceInternal(final long offset,
             if (!entry.getKey().equals(orderedBlockList.get(mapEntry))) {
               LOG.debug("The order for the given offset {} with blockId {} " +
                       " for the path {} was not successful", offset, entry.getKey(), path);
-              throw new IOException("The ordering in map is incorrect");
+              throw new IOException("The ordering in map is incorrect for blockId " +
+                      entry.getKey() + " and offset " + offset + " for path" + path);
```

Comment:
StreamId here as well

Reply:
In other places where exception is thrown as well.

Reply:
Added